arkane/orca.py

```diff
@@ -115,30 +115,30 @@ def load_geometry(self):
         for i in reversed(range(len(log))):
             line = log[i]
             if 'CARTESIAN COORDINATES (ANGSTROEM)' in line:
-                for line in log[(i + 3):]:
-                    if not line.strip():
+                for coordinates in log[(i + 2):]:
```

Comment:
I think this should remain `line` - otherwise it'll be confusing for someone else to read this code, they would expect coordinates to contain actual coordinates (list or tuple), and not a string with additional info